app/Exports/ScheduleExport.php

```diff
@@ -27,6 +29,7 @@ public function view(): View
             'program' => $this->program,
             'position' => $this->position,
             'workspace' => $this->workspace,
+            'specialDays' => $this->specialDays
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Exports/ScheduleExport.php</br>**Line**: 32</br>Each line in an array declaration must end in a comma